fixtures/unused-receiver.go

```diff
@@ -0,0 +1,18 @@
+package fixtures
+
+import (
+	"fmt"
+
+	"github.com/mgechev/revive/lint"
+)
+
+func (f *Unix) Name() string { // MATCH /method receiver 'f' is not referenced in method's body, consider removing or renaming it as _/
```

Comment:
Can we also add a test case in which we have local variable named after the receiver and we use it?

Also one case when the receiver is used.